include/layers/EWLayer.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include <algorithm>
+
+#include "layers/FCLayer.hpp"
+
+template <typename ValueType>
+class EWLayer : public Layer<ValueType> {
```

Comment:
one of the variant inherit from EWLAyer for example SinEWLayer and etc

Reply:
discussed offline, not needed